releasenotes/notes/obfuscate-json-operators-postgres-9d53904d3eb9f9ed.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add support for postgres json operators in sql obfuscate package.
+fixes:
+  - |
+    Fixes an issue where certain (#> and #>>) postgres json operators were
+    being interpreted as comments and removed in obfuscate package.
```

Comment:
```suggestion
    being interpreted as comments and removed by the obfuscate package.
```

Reply:
Not sure if this one should be "by the" or "from the" ?

Reply:
I think "by the" makes the most sense.